components/builder-protocol/protocols/originsrv.proto

```diff
@@ -31,6 +31,7 @@ message Origin {
   optional string name = 2;
   optional uint64 owner_id = 3;
   optional string private_key_name = 4;
+  optional uint64 unique_package_count = 5;
```

Comment:
A couple of comments - this seems a bit out of place in Origin definition - I'm not sure this is the way we want to go. Can we look at adding to the stats API that has package count already instead?

Reply:
My first thought was the same as yours - does this belong in the Origin definition?  Seeing `private_key_name` in there made me think that we weren't being super strict about what went into the Origin definition.  The goal here was to give @elliott-davis the info he needed with a minimum of API calls.  I'll check out the stats API.

Reply:
Yeah I'm not sure private_key_name should be there either :(